synfig-studio/src/gui/widgets/widget_coloredit.cpp

```diff
@@ -287,26 +294,55 @@ ColorSlider::on_event(GdkEvent *event)
 }
 
 /* === M E T H O D S ======================================================= */
+void 
+Widget_ColorEdit::SliderRow(int i,ColorSlider * n, char * l, Pango::AttrList & attr_list, Gtk::Table* table)
+{
+	Gtk::Label *label;
+	n->signal_slider_moved().connect(sigc::mem_fun(*this,&studio::Widget_ColorEdit::on_slider_moved));
+	//n->signal_activated().connect(sigc::mem_fun(*this,&studio::Widget_ColorEdit::activated));
+	n->signal_activated().connect(sigc::mem_fun(*this,&studio::Widget_ColorEdit::on_value_changed));
+	label=manage(new class Gtk::Label(l,0.0,0.5));
+	label->set_use_markup(false);
+	label->set_use_underline(false);
+	label->set_attributes(attr_list);
+	table->attach(*label, 0, 1, 1+2*i, 2+2*i, Gtk::EXPAND|Gtk::FILL, Gtk::EXPAND|Gtk::FILL, 0, 0);
+	table->attach(*n, 0, 1, 2+2*i, 3+2*i, Gtk::EXPAND|Gtk::FILL, Gtk::EXPAND|Gtk::FILL, 0, 0);
+}
+
+void
+Widget_ColorEdit::AttachSpinButton(int i, Gtk::SpinButton * n, Gtk::Table * table)
+{
+	n->set_update_policy(Gtk::UPDATE_ALWAYS);
+	n->set_size_request(SPINBUTTON_WIDTH,-1);
+	n->set_range(0.0,100.0);
```

Comment:
We need the possibility to set colors outside of the range [0,100].
For example: (200, 200, 200) - "twice white" HDRI color.